Services/Store/ModsenOnlineStore.Store.Application/Interfaces/CommentInterfaces/ICommentService.cs

```diff
@@ -5,14 +5,14 @@ namespace ModsenOnlineStore.Store.Application.Interfaces.CommentInterfaces
 {
     public interface ICommentService
     {
-        Task<ResponseInfo<List<GetCommentDto>>> GetAllComments();
+        Task<ResponseInfo> GetAllComments();
```

Comment:
why are there just reponse info, when this method must return data?

Reply:
We decided to inherit DataResponseInfo from ResponseInfo. That's why all methods return Task<ResponseInfo>

Reply:
Yeah, but how I can get data from controller methods. Lets say I need to check some property in comment. So if you return just ResponseInfo I need to know that I can cast it to DataResponseInfo and eventually do it, which is redundant operation.
Even worse if I do not know that there is such class like DataReponseInfo, then I won't know how to get this comments at all.

Reply:
got it, fixed